src/gobupload/relate/update.py

```diff
@@ -192,8 +185,8 @@ def create_event(self, row: dict):
                 FIELD.HASH: row[FIELD.HASH],
             }
             return MODIFY.create_event(row['rel_id'], data, _RELATE_VERSION)
-        data = {FIELD.LAST_EVENT: row[FIELD.LAST_EVENT]}
-        return CONFIRM.create_event(row['rel_id'], data, _RELATE_VERSION)
+
+        return None  # No event created, CONFIRMS are skipped
```

Comment:
Shouldn't we just replace this with a BULKCONFIRM instead?

Reply:
We were not applying (and storing) the CONFIRM events due to this code:
```
        if catalogue != "rel":
            _apply_confirms(storage, confirms, timestamp=timestamp, stats=stats)
```
In that case we can skip the writing to a temp file (ConfirmsWriter) I think?

Reply:
Ah that's weird. You're right. In that case it's useless to pass them on ;) That was probably a performance optimisation in the past as well; ignoring 'rel' confirms. But yeah, no need to create them then. Got my approval!